core/src/main/java/tc/oc/pgm/spawns/SpawnMatchModule.java

```diff
@@ -77,8 +76,7 @@ public Match getMatch() {
 
   public RespawnOptions getRespawnOptions(Query query) {
     return module.respawnOptions.stream()
-        .filter(
-            respawnOption -> respawnOption.filter.query(query).equals(Filter.QueryResponse.ALLOW))
+        .filter(respawn -> !respawn.filter.query(query).isDenied())
```

Comment:
This seems to now allow filters abstaining, I can't think of any reason right now why that would be bad for future xmls but it does potentially change behaviour on existing maps?

Reply:
The modification makes no behavior change. I considered doing isAllowed, but that does imply a behavior change (abstaining used to be removed, now they would be accepted) which is why i made it not-denied (which is exactly the same behavior as current)

Reply:
Not sure I understand, 

(From Filter.java)
```java
public boolean isDenied() {
      return this == DENY;
}
```
This implies that ABSTAIN will pass right? As `!DENY == ALLOW || ABSTAIN`

Reply:
Ignore everything i said, i was under the impression isAllowed was `allow || abstain`, and isDenied was `deny || abstain`, which does not seem to be the case at all.

I'll revert to keep the behavior unmodified